change-validation/src/main/scala/bootstrap/rudder/plugin/ChangeValidationConf.scala

```diff
@@ -134,13 +135,32 @@ class ChangeValidationWorkflowLevelService(
    * Methode to use to combine several validationNeeded check. It's the same for all objects?
```

Comment:
you should probably add a comment there saying that it always skips workflow for validated user